crates/symbolicator-service/src/services/symbolication/js.rs

```diff
@@ -38,29 +38,27 @@ impl SymbolicationActor {
         let stacktraces: Vec<_> = request
             .stacktraces
             .into_iter()
-            .map(|trace| async {
-                js_processing_symbolicate_stacktrace(trace, &sourcemap_lookup).await
-            })
+            .map(|trace| async { symbolicate_js_stacktrace(trace, &sourcemap_lookup).await })
             .collect();
 
         let stacktraces = futures::future::join_all(stacktraces).await;
```

Comment:
no need to `collect` here btw, `join_all` takes an `IntoIterator`

Reply:
TIL!